### PR TITLE
Added scheduler section in the tutorial

### PR DESCRIPTION
**Pull request by gemblerz**

### CODE REVIEW

File: docs/tutorials/schedule-jobs.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Job Starttime: 2022-10-10 02:21:37.373437 +0000 UTC
+
+===== SCHEDULING DETAILS =====
+Science Goal ID: 45afe963-5b8b-4e15-654c-54e2946f2ddb
+Total number of nodes 1
+```
+
+The job status can be also shown in [job status page](https://portal.sagecontinuum.org/job-status).
+
+## Access to data
```
Comment (jswantek) [inline docs/tutorials/schedule-jobs.md:95]: please add a link to the "access data" page here for more details

File: docs/tutorials/schedule-jobs.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+ "job_id": "56",
+ "status": "Submitted"
+}
+```
+
+:::note
+If you receive any error related to permission on nodes or scheduling, please contact us.
+:::
+
+## Check status of job
```
Comment (jswantek) [inline docs/tutorials/schedule-jobs.md:77]: Please add how to see all the jobs that are submitted for a user

```
sesctl stat
```

File: docs/tutorials/schedule-jobs.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+successcriteria:
+- WallClock(1d)
+EOF
+```
+
+We attempt to run a plugin named `image-sampler` to collect an image from the camera named `bottom` on `W030` node. As a result of the run, we will get images from the job. The job also specifies that the plugin needs to be scheduled every minute (i.e., `* * * * *` in [crontab expression](https://crontab.guru/)). The job completes 24 hours after the job started to run on the node.
+
+## Submit the job using _sesctl_
+
+`sesctl` is a command-line tool to manage jobs exist in the scheduler.
```
Comment (jswantek) [inline docs/tutorials/schedule-jobs.md:47]: How do they get this tool on their machine?  link to the github project where they can clone / access it?

File: docs/tutorials/schedule-jobs.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+===== SCHEDULING DETAILS =====
+Science Goal ID: 45afe963-5b8b-4e15-654c-54e2946f2ddb
+Total number of nodes 1
+```
+
+The job status can be also shown in [job status page](https://portal.sagecontinuum.org/job-status).
+
+## Access to data
+
+A few minutes later, the `W030` Waggle node would start collecting images by scheduling the plugin on the node.
```
Comment (jswantek) [inline docs/tutorials/schedule-jobs.md:97]: .. and those images will be published to the Beehive(add link here to the architecture page) and available for download.

**Review by jswantek — CHANGES_REQUESTED** [review]

File: docs/tutorials/schedule-jobs.md (hunk 1 above)
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:95]: Done!

File: docs/tutorials/schedule-jobs.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 5
+---
+
+# Submit your job
+
+Are you ready to deploy your plugins to measure the world? We will use [edge scheduler](../about/architecture.md#edge-scheduler-es) to submit a job to demonstrate how you can deploy plugins to field-deployed Waggle nodes.
+
+:::caution
+If you have not created your account, please go to [here](https://access.sagecontinuum.org) and log in to create a new account with your email. Once logged in, you will be able to create and edit your jobs, but will need a permission to submit jobs to the scheduler. Please contact us at [contact-us](../contact-us.md) to request the permission for job submission.
```
[the remark this answers is not in the document]
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:10]: I added a few sections for the job submission procedure.

File: docs/tutorials/schedule-jobs.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 5
```
[the remark this answers is not in the document]
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:2]: Done!

File: docs/tutorials/schedule-jobs.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+---
+
+# Submit your job
```
[the remark this answers is not in the document]
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:5]: We have `## Upload your job to the scheduler` section that describes how to create a job in the scheduler.

File: docs/tutorials/schedule-jobs.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+Are you ready to deploy your plugins to measure the world? We will use [edge scheduler](../about/architecture.md#edge-scheduler-es) to submit a job to demonstrate how you can deploy plugins to field-deployed Waggle nodes.
+
+:::caution
+If you have not created your account, please go to [here](https://access.sagecontinuum.org) and log in to create a new account with your email. Once logged in, you will be able to create and edit your jobs, but will need a permission to submit jobs to the scheduler. Please contact us at [contact-us](../contact-us.md) to request the permission for job submission.
+:::
+
+A job describes,
+- plugins that are registered and built in [edge code repository](../about/architecture.md#edge-code-repository-ecr) with specification including any plugin arguments,
```
[the remark this answers is not in the document]
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:14]: Done

File: docs/tutorials/schedule-jobs.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+- a list of Waggle nodes on which the plugins will be run,
```
[the remark this answers is not in the document]
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:15]: That item talks about nodes. So, the word "scheduled" isn't important to say, but I changed it to "will be scheduled and run".

File: docs/tutorials/schedule-jobs.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+- science rules describing when the plugins should be scheduled,
+- conditions to determine when the job completes
+
+## Create a job
+
+We create a job in YAML format (JSON format is also supported. Please check out details of job attributes [here](https://github.com/waggle-sensor/edge-scheduler/tree/main/docs/sesctl).)
```
[the remark this answers is not in the document]
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:21]: Done

File: docs/tutorials/schedule-jobs.md (hunk 3 above)
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:47]: Done

File: docs/tutorials/schedule-jobs.md (hunk 4 above)
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:97]: Done

File: docs/tutorials/schedule-jobs.md (hunk 2 above)
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:77]: It is described on #Upload your job to the scheduler section.

File: docs/tutorials/schedule-jobs.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 5
+---
```
[the remark this answers is not in the document]
Comment (gemblerz) [reply docs/tutorials/schedule-jobs.md:3]: The science goal in the architecture page is defined as a Job. I describe that a job is an instance of science goal.

Also removed the word "application".

**Pull request merged**